deploy/node_policy.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS node_policy(
+    node_id CHAR(32),
+    name VARCHAR(256),
+    policy_group VARCHAR(256),
+    CONSTRAINT node_policy_node_id_fkey
+      FOREIGN KEY(node_id)
+      REFERENCES nodes(id)
```

Comment:
Should deletes to the node cascade here? If not, are there pedant tests to define the behavior if you have a node_policy for a node that doesn't exist?